.aws/task_definition.json

```diff
@@ -1,14 +1,13 @@
 {
-    "taskDefinitionArn": "arn:aws:ecs:eu-west-3:253053805092:task-definition/quivr:160",
+    "taskDefinitionArn": "arn:aws:ecs:us-west-1:155722108667:task-definition/sr-vt-prod-api:2",
     "containerDefinitions": [
         {
-            "name": "quivr",
-            "image": "253053805092.dkr.ecr.eu-west-3.amazonaws.com/quivr:bada136312ad3497664c3562a36b263d43c89c53",
-            "cpu": "1024",
-            "memory": "2048",
+            "name": "sr-vt-prod-api",
```

Comment:
we decided to call this `sr-vt-prod-backend`? 
`sr-vt-prod-api` is fine, but please purpose the name change easier in the future.